packages/core/src/app/shipping/hooks/useMultishippingConsignmentItems.ts

```diff
@@ -13,6 +14,27 @@ const calculateShippableItemsCount = (items: MultiShippingTableItemWithType[]):
     return items.reduce((total, item) => total + item.quantity, 0);
 };
 
+const hasSplitItem = (
+    items: MultiShippingTableItemWithType[],
+    itemHashMap: Map<string, string>,
+  ): boolean => {
+    const processedHashes = new Set<string>();
+  
+    return items.some((item) => {
+      const hash = itemHashMap.get(item.id.toString());
+  
+      if (hash && processedHashes.has(hash)) {
+        return true;
+      }
+  
+      if (hash) {
+        processedHashes.add(hash);
+      }
+  
+      return false;
```

Comment:
We may want to avoid seeing two `if(hash)` in a row.
```suggestion
    if (!hash) {
      return false;
    }

    if (processedHashes.has(hash)) {
      return true;
    }

    processedHashes.add(hash);
    return false;
```